source/_integrations/iotawatt.markdown

```diff
@@ -23,7 +23,11 @@ and create them as sensors in Home Assistant.
 
 ## Energy management
 
-You can use the accumulated energy sensors directly with the Home Assistant energy dashboard.
+You can use the energy sensors directly with the Home Assistant energy dashboard.
+
+<div class="note warning">
+The Accumulated sensors should not be used.  That was a legacy feature for older versions of IotaWatt before it had integrators.  They should be removed in a future version of this integration.  
+</div>
```

Comment:
I don't follow this addition, if an sensor that is invalid show up for use in the energy dashboard, while it shouldn't: That would be a bug.

We don't document bug.

Reply:
I'm working on a PR to remove the bad sensors.  That will be a breaking change for anyone who setup their IotaWatt a long time ago, before the integrators feature was added.  I guess this explanation belongs in the release notes for that, so I'll remove it here.